pkgs/applications/editors/emacs/elisp-packages/manual-packages/gn-mode/package.nix

```
@@ -0,0 +1,18 @@
+{
+  gn,
+  melpaBuild,
+}:
+
+melpaBuild {
+  pname = "gn-mode";
+  inherit (gn) src version;
```

`gn.version` is "2020-03-09", but Emacs wants "2020.03.09".

```suggestion
  inherit (gn) src;
  version = "0-unstable-${gn.version}";
```